### PR TITLE
Publish cluster state update stats

### PR DESCRIPTION
**Pull request by amathur1893**
This PR will start publishing latency and failure metrices for Publish Phase of Cluster update from Master's Persepctive. This is going to help us in RCA and live time tracking of issues where publication is failing again and agian and the cluster is unstable because of this.

1.Tested using Docker

Tmp file

^master_cluster_update
{"current_time":1614584101385}
{"PublishClusterState_Failure":2,"PublishClusterState_Latency":0}$

Table created

sqlite> .tables
PublishClusterState_Latency 
PublishClusterState_Failure

Contents of the table

sqlite> select * from PublishClusterState_Failure;
6.0|6.0|6.0|6.0
sqlite> select * from PublishClusterState_Latency;
29.0|29.0|29.0|29.0

### CODE REVIEW

File: src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rca/framework/metrics/ExceptionsAndErrors.java [hunk 1]
```diff
@@ -59,6 +59,8 @@
 
   CLUSTER_APPLIER_SERVICE_STATS_COLLECTOR_ERROR("ClusterApplierServiceStatsCollector"),
 
+  MASTER_CLUSTER_UPDATE_STATS_COLLECTOR_ERROR("MasterClusterStateUpdateStatsCollector"),
```
Comment (yu-sun-77) [inline src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rca/framework/metrics/ExceptionsAndErrors.java:62]: Can we rename it to "MasterClusterStateUpdateStatsCollectorError"?

Comment (yu-sun-77) [reply src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rca/framework/metrics/ExceptionsAndErrors.java:62]: Other constants have been taken care in the other PR:
https://github.com/opendistro-for-elasticsearch/performance-analyzer-rca/pull/583/files

**Review by yu-sun-77 — COMMENTED** [review]
Can we add description of this change?

**Review by yu-sun-77 — DISMISSED** [review]

File: src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/metrics/AllMetrics.java [hunk 2]
```diff
@@ -845,6 +846,27 @@ public String toString() {
     }
   }
 
+  public enum MasterClusterUpdateStatsValue implements MetricValue {
+    PUBLISH_CLUSTER_STATE_LATENCY(Constants.PUBLISH_CLUSTER_STATE_LATENCY),
+    PUBLISH_CLUSTER_STATE_FAILURE(Constants.PUBLISH_CLUSTER_STATE_FAILURE);
```
Comment (khushbr) [inline src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/metrics/AllMetrics.java:851]: Good job naming the variables!

**Review by khushbr — APPROVED** [review]

**Review by khushbr — APPROVED** [review]

**Review by khushbr — APPROVED** [review]

**Review by meetshah777 — APPROVED** [review]

**Review by meetshah777 — APPROVED** [review]

**Pull request merged**